### PR TITLE
add new task for Pb-Pb multiplicity

### PR DESCRIPTION
**Pull request by hhesouno**
Based on O2 tutorial vol.2 I created a simple task for making basic histograms that will be used for multiplicity analysis of Pb-Pb collisions from Run 3.

### CODE REVIEW

File: PWGMM/Tasks/multiplicityPbPb.cxx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+using namespace o2::framework;
+using namespace o2::framework::expressions;
+
+struct myExampleTask {
+
+  // Configurable<float> estimatorEta{"estimatorEta", 1.0, "eta range for INEL>0 sample definition"};
+
+  HistogramRegistry histos{"histos", {}, OutputObjHandlingPolicy::AnalysisObject};
+
+  Configurable<int> nBinsEta{"nBinsEta", 100, "N bins in eta histo"};
```
Comment (aalkin) [inline PWGMM/Tasks/multiplicityPbPb.cxx:33]: This is a minor comment, but you may want to consider using `ConfigurableAxis` that is more flexible.

File: PWGMM/Tasks/multiplicityPbPb.cxx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+#include "Common/DataModel/TrackSelectionTables.h"
+#include "Framework/ASoAHelpers.h"
+
+#include "ReconstructionDataFormats/GlobalTrackID.h"
+
+using namespace o2;
+using namespace o2::framework;
+using namespace o2::framework::expressions;
+
+struct myExampleTask {
```
Comment (aalkin) [inline PWGMM/Tasks/multiplicityPbPb.cxx:27]: Please rename the task to something more meaningful.

File: PWGMM/Tasks/multiplicityPbPb.cxx [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+    // histos.fill(HIST("Multiplicity"), groupedTracks.size());
+
+    histos.fill(HIST("Anton/ZvtxEvents"), collision.posZ());
+
+    for (auto& track : tracks) {
+      if (track.tpcNClsCrossedRows() < 70)
+        continue; // badly tracked
+
+      trackCounter = 1 + trackCounter;
```
Comment (aalkin) [inline PWGMM/Tasks/multiplicityPbPb.cxx:102]: This can also be done with `++trackCounter;`.

**Review by aalkin — CHANGES_REQUESTED** [review]

File: PWGMM/Tasks/multiplicityPbPb.cxx (hunk 2 above)
Comment (aalkin) [reply PWGMM/Tasks/multiplicityPbPb.cxx]: Please also change the task type name in line 123 from `adaptAnalysisTask<myExampleTask>(cfgc)` to `adaptAnalysisTask< multiplicityPbPb >(cfgc)`.

**Review by aalkin — APPROVED** [review]

File: PWGMM/Tasks/multiplicityPbPb.cxx (hunk 1 above)
Comment (hhesouno) [reply PWGMM/Tasks/multiplicityPbPb.cxx:33]: Is there a documentation, how it is used? I could not find one.

**Pull request merged**